src/main/webapp/newuser.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
```

Comment:
I was wondering why you're choosing to call this "newuser.html", since it looks like this page allows a user to either join an existing room or create a new room, so I don't think it has to do with creating a new user.

I think a better name might be "room.html" or "join-room.html". What do you think?

The same applies to the JS file as well

Reply:
@pruess-google yes I thinking of it as the user either joining or creating a room which makes them a new user, but that doesn't really make any sense. But join-room makes a lot of sense, I changed it!